api_types.ts

```diff
@@ -526,6 +526,14 @@ export interface FetchRequest {
    * wish to make an unauthenticated supporting request as part of a formula implementation.
    */
   disableAuthentication?: boolean;
+  /**
+   * Whether to follow the redirects indicated by an HTTP 301 or 302 response
+   * and fetch the URL indicated by the redirect. If false, requests returning
+   * an HTTP 301 or 302 response will be returned as-is, and you may inspect the
+   * `Location` header of the response to observe the indicated redirect URL.
+   * Defaults to true if unset.
+   */
+  followRedirects?: boolean;
```

Comment:
Would changing this to `ignoreRedirects` help with both the double negatives we need to deal with on the coda side as well as make the default value of `undefined` === "don't ignore redirects" be the expected default behavior?

Reply:
Sure, I like it.